### PR TITLE
ICMSLST-701 Refactor Importer/Exporter Access Request workbasket code.

### PR DESCRIPTION
**Pull request by MattHolmes123**
Move logic from templates to python model.

### CODE REVIEW

File: web/templates/web/domains/workbasket/workbasket.html [hunk 1]
```diff
@@ -141,15 +141,15 @@ <h4>
 
                 <td>{{ row.subject }}</td>
 
-                <td>{{ row.company }}</td>
+                <td>{{ row.company|safe }}</td>
```
Comment (kevinetienne) [inline web/templates/web/domains/workbasket/workbasket.html:144]: should safe be [`nl2br`](https://github.com/uktrade/icms/blob/develop/web/jinja2.py#L17-L25)?  Same comment apply for other change in this file.

Comment (kevinetienne) [reply web/templates/web/domains/workbasket/workbasket.html:144]: Do we capture html for the `company` and `info`

Comment (MattHolmes123) [reply web/templates/web/domains/workbasket/workbasket.html:144]: I will have a look and more than likely change it to use nl2br

**Review by kevinetienne — APPROVED** [review]

File: web/domains/case/access/models.py [hunk 2]
```diff
@@ -63,25 +62,99 @@ class AccessRequest(WorkbasketBase, Process):
...
+                    name="View",
+                    url=reverse(
+                        "case:view", kwargs={"application_pk": self.pk, "case_type": "access"}
+                    ),
+                )
+            ]
+
+            for fir in self.further_information_requests.open():
+                kwargs = {"application_pk": self.pk, "fir_pk": fir.pk, "case_type": "access"}
+                owner_actions.append(
```
Comment (oskusalerma) [inline web/domains/case/access/models.py:145]: empty line before

File: web/domains/case/access/models.py [hunk 3]
```diff
@@ -63,25 +62,99 @@ class AccessRequest(WorkbasketBase, Process):
...
+            info_rows.append("Approval Requested")
+
+        r.information = "\n".join(info_rows)
+
+        task = self.get_active_task()
+
+        if user.has_perm("web.reference_data_access"):
+            admin_actions: list[WorkbasketAction] = []
+
+            if self.process_type == "ExporterAccessRequest":
```
Comment (oskusalerma) [inline web/domains/case/access/models.py:108]: how about doing this:

```
if self.process_type == "exporter...":
  entity = "exporter"
else:
  entity = "importer"

```  
and then this collapses down to just one admin_actions.append

**Review by oskusalerma — CHANGES_REQUESTED** [review]

**Review by oskusalerma — APPROVED** [review]

**Pull request merged**